dir.props

```diff
@@ -109,7 +109,7 @@
     <NativeBinDir>$(BinDir)$(OSGroup).$(ArchGroup).$(ConfigurationGroup)/native</NativeBinDir>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetGroup)' == 'netfx' AND '$(IsTestProject)' != 'true'">
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netfx' AND '$(IsTestProject)' != 'true' AND '$(BuildNet462AndNewer)' != 'false'">
```

Comment:
Actually nevermind, you want to keep that one so that we don't break the scenario of building individual projects for `/p:TargetGroup=netfx`